apps/web/src/pages/Home/HomePage/components/Opportunities/Opportunities.tsx

```diff
@@ -82,95 +75,79 @@ const Opportunities = () => {
         <CardFlat
           color="secondary"
           title={intl.formatMessage({
-            defaultMessage: "IT Apprenticeship Program for Indigenous Peoples",
-            id: "4N/PxH",
-            description:
-              "Heading for the IT Apprenticeship Program for Indigenous Peoples on home page",
+            defaultMessage: "Managers community",
```

Comment:
@BrookeCameron are you sure you want this plural?  "Executive community" is singular.  Or did you want the possesive "Manager's community"?